### ISSUE TITLE
support Private DNS View and Zone

### ISSUE THREAD

**Issue by chanstev** [oci-landing-zones/terraform-oci-modules-networking#40]
[reference implementation ](https://github.com/oci-landing-zones/terraform-oci-multicloud-azure/tree/main/modules/oci-network-dns)

**Comment by andrecorreaneto** [comment]
Hi @chanstev, the module does support private DNS views and zones. See 
https://github.com/oci-landing-zones/terraform-oci-modules-networking/blob/main/variables.tf#L225
and
https://github.com/oci-landing-zones/terraform-oci-modules-networking/blob/main/dns.tf

**Comment by VinayKumar611** [comment]
Hi @chanstev , 

Thanks for your feedback. We assume the relevant information will be enough to unblock you. We are closing the ticket now, if needed please create the new ticket.

Regards, 
Vinay Kumar
